### PR TITLE
Add lumi vs. mahti info and lumi-link to nav

### PR DESCRIPTION
**Pull request by rkronberg**
https://csc-guide-preview.rahtiapp.fi/origin/lumi-info/computing/lumi-vs-mahti/

### CODE REVIEW

**Comment by rkronberg** [comment]
Asked Emmanuel also to review the page. The link to this page is now under https://csc-guide-preview.rahtiapp.fi/origin/lumi-info/computing/available-systems/#lumi but should it be also somwhere else?

File: docs/computing/lumi-vs-mahti.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# How does LUMI-C differ from Mahti?
+
+[LUMI-C](https://docs.lumi-supercomputer.eu/computing/systems/lumic/) consists of 1536 AMD CPU nodes (2x64 cores each) with a performance of 5.6 petaflops. The node count of [Mahti](systems-mahti.md) is similar, 1404 AMD CPU nodes (2x64 cores each) corresponding to a performance of 7.5 Petaflops. Although the systems are very alike based on hardware, core count and performance, there are important differences related to access and use which are highlighted on this page.
```
Comment (galfthan) [inline docs/computing/lumi-vs-mahti.md:3]: 5.6 is linpack for Lumi, and 7.5 is theoretical for Mahti CPU nodes (1404 * 128 * 2.6 * 16). Theoretical for Lumi-C is 7.7 Petaflops (1536 * 128 * 2.45 * 16)

Comment (galfthan) [reply docs/computing/lumi-vs-mahti.md:3]: And perhaps that Mahti has small amount of GPU nodes (24) and Lumi-C will be flanked by Lumi-D and Lumi-G with huge GPU capacity. 

Also, Mahti has 256GB memory in all CPU nodes, while Lumi-C in Puhti like fashion has also 512 and 1024 GiB nodes

File: docs/computing/lumi-vs-mahti.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Accessing and SSH keys
+
+To access LUMI, you need to first [create a LUMI-specific project](../accounts/how-to-create-new-project.md#how-to-create-finnish-lumi-projects). Furthermore, accessing LUMI is only possible using SSH keys, meaning that you cannot use passwords to connect through SSH like on Mahti. For instructions on how to generate an SSH key pair and uploading the public key to [MyCSC](https://my.csc.fi/), see [Setting up SSH keys](connecting.md#setting-up-ssh-keys) and the [Get started with LUMI](https://docs.lumi-supercomputer.eu/firststeps/getstarted/) pages.
+
+## Finite time projects
```
Comment (galfthan) [inline docs/computing/lumi-vs-mahti.md:9]: Not only that they have finite time, but in general that they are a different type of projects, and cannot use same projects for lumi and national stuff. Both available from my.csc.fi though

File: docs/computing/lumi-vs-mahti.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+|Benchmark    |3 months|No                             |
+|Extreme scale|1 year  |No                             |
+|Development  |1 year  |Yes, twice = max. 3 years total|
+
+For more details on the access modes, see the [LUMI access page on the Services for Research website](https://research.csc.fi/lumi-access). Note that LUMI users from Finland are also eligible to apply for EuroHPC Joint Undertaking (JU) resources. [See more details on European access modes here](https://www.lumi-supercomputer.eu/get-started-2021/users-in-europe/).
+
+## Software installation policy
+
+While Mahti offers frequently used applications as pre-installed modules, users are expected to compile and install the applications they intend to run on LUMI themselves. This is due to the fact that LUMI is a pan-European system with significantly more users than the national systems. Maintaining a pre-installed collection of software and licenses that would meet all users' needs on a European level would thus require excessive administration efforts. Moreover, neither the local organizations such as CSC nor the LUMI user support team (LUST) have root access to the system.
```
Comment (galfthan) [inline docs/computing/lumi-vs-mahti.md:24]: I would just state that it is like this right now, and not here make so much the argument why.  The root access stuff is at least not a real reason, we do not have root on puhti and mahti either.

Comment (galfthan) [reply docs/computing/lumi-vs-mahti.md:24]: and cops does have root to lumi of course

File: docs/computing/lumi-vs-mahti.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+!!! Note
+    Irrespective of the loaded compiler suite, one noticeable difference concerning the LUMI-C programming environment is that it comes with compiler wrappers that replace commands commonly found on HPC systems such as Mahti. For example, commands for compiling MPI code like `mpicc`, `mpic++` and `mpif90` are unavailable as such. Instead, you should use the wrappers `cc`, `CC` and `ftn`, respectively. See the LUMI documentation for [more details on the available MPI wrappers](https://docs.lumi-supercomputer.eu/development/compiling/prgenv/#compile-an-mpi-program).
+
+## Disk areas
+
+Similar to CSC supercomputers, LUMI uses the Lustre parallel filesystem. However, there's currently no fast local disk such as on Puhti and Mahti-AI. The fast flash-based scratch space (LUMI-F) will be made available once the installation of LUMI-G is completed. Also, the object storage (LUMI-O) is not available yet. [See the LUMI documentation for more details](https://docs.lumi-supercomputer.eu/storage/).
```
Comment (galfthan) [inline docs/computing/lumi-vs-mahti.md:39]: "Currrently no", I do not think it is planned either?

Comment (galfthan) [reply docs/computing/lumi-vs-mahti.md:39]: Though I recently noticed Frontier has 2x2TiB in all nodes

File: docs/computing/lumi-vs-mahti.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+## Available partitions
+
+LUMI-C has two types of partitions (queues): two that are allocatable by node (only full nodes can be requested, similar to Mahti) and three that are allocatable by resources (partial nodes can be requested, similar to Puhti). [See more details in the LUMI documentation](https://docs.lumi-supercomputer.eu/computing/jobs/partitions/), e.g. maximum wall-time/node count and naming of the partitions.
+
+## Billing
+
+Billing on LUMI-C differs from Mahti, as well as depending on which partition you are using. Notably, the standard partition (allocatable by node) on LUMI-C is billed based on reserved CPU-core hours (128 billing units per node hour), while on Mahti reserving one node consumes 100 BUs per hour. The small partition on LUMI-C (allocatable by core) is on the other hand billed either 1) per reserved CPU-core (1 BU per CPUh), or 2) per allocated chunk of 2 GB of memory _if_ your are using more than 2 GB of memory per core. [See the LUMI documentation for more details and precise formulas](https://docs.lumi-supercomputer.eu/computing/jobs/billing/).
```
Comment (galfthan) [inline docs/computing/lumi-vs-mahti.md:47]: I would not explain the model so much here, just say it is different and give link to billing.  A key difference that should be mentioned is that there are three currencies. GPU hours, cpu hours, and storage hours. This is a key difference to having only one BU

File: docs/computing/lumi-vs-mahti.md (hunk 4 above)
Comment (galfthan) [reply docs/computing/lumi-vs-mahti.md:39]: The or a? It is not Mahtis :)

**Review by galfthan — CHANGES_REQUESTED** [review]

File: docs/computing/lumi-vs-mahti.md (hunk 4 above)
Comment (rkronberg) [reply docs/computing/lumi-vs-mahti.md:39]: > "Currrently no", I do not think it is planned either?

Maybe I misunderstood. LUMI-F will be made available though, but I guess it is not a fast local disk, but a fast flash-based storage space? Is this what you mean?

Comment (galfthan) [reply docs/computing/lumi-vs-mahti.md]: Exactly, it is flash based lustre file system, not local disk.  https://docs.lumi-supercomputer.eu/storage/parallel/lumif/

**Review by galfthan — APPROVED** [review]

**Review by attesillanpaa — APPROVED** [review]

**Pull request merged**